.github/workflows/build.yml

```diff
@@ -199,7 +202,7 @@ jobs:
 
     - name: Build C library
       if: ${{ !contains(matrix.os, 'windows') }} # Requires an additional adapted Makefile for `cl.exe` compiler
-      run: make.sh CFLAGS="-Werror" -j
```

Comment:
Just one note, can you bring this back here and remove "IS_WINDOWS"? currently make.sh only runs on Linux anyways

Reply:
It's ok, in this way it works for build.rs too